src/main/java/net/minestom/server/network/packet/PacketRegistry.java

```diff
@@ -364,7 +430,7 @@ protected PacketInfo<T> computeValue(@NotNull Class<?> type) {
                         return (PacketInfo<T>) info;
                     }
                 }
-                throw new IllegalStateException("Packet type " + type + " isn't registered!");
+                throw new IllegalStateException("Packet type " + type + " isn't registered for state " + side().name() + "_" + state().name() + "!");
```

Comment:
This should probably be rephrased. `packet type {type} cannot be sent in state {side} {state}`